Argh, why is there now an RDKit failure? [TestRDKitConverter.test_sanitize_fail_warning](https://github.com/MDAnalysis/mdanalysis/actions/runs/3649646773/jobs/6164593327#step:7:521):
```
FAILED testsuite/MDAnalysisTests/converters/test_rdkit.py::TestRDKitConverter::test_sanitize_fail_warning - UserWarning: NaN detected in coordinates, the output molecule will not have 3D coordinates assigned
```

--- a/testsuite/MDAnalysisTests/converters/test_rdkit.py
+++ b/testsuite/MDAnalysisTests/converters/test_rdkit.py
@@ -398,7 +398,8 @@ class TestRDKitConverter(object):
         with pytest.warns(UserWarning, match="Could not sanitize molecule"):
             u.atoms.convert_to.rdkit(NoImplicit=False)
         with warnings.catch_warnings():
-            warnings.simplefilter("error")
+            warnings.filterwarnings("error", "Could not sanitize molecule")
+            warnings.warn("Could not sanitize molecule: failed during step")
             u.atoms.convert_to.rdkit()
 
 

--- a/testsuite/MDAnalysisTests/converters/test_rdkit.py
+++ b/testsuite/MDAnalysisTests/converters/test_rdkit.py
@@ -399,7 +399,6 @@ class TestRDKitConverter(object):
             u.atoms.convert_to.rdkit(NoImplicit=False)
         with warnings.catch_warnings():
             warnings.filterwarnings("error", "Could not sanitize molecule")
-            warnings.warn("Could not sanitize molecule: failed during step")
             u.atoms.convert_to.rdkit()
 
 

--- a/testsuite/MDAnalysisTests/converters/test_rdkit.py
+++ b/testsuite/MDAnalysisTests/converters/test_rdkit.py
@@ -400,9 +400,6 @@ class TestRDKitConverter(object):
         with warnings.catch_warnings():
             warnings.simplefilter("error")
             u.atoms.convert_to.rdkit()
-        if record:
-            assert all("Could not sanitize molecule" not in str(w.message)
-                       for w in record.list)
 
 
 @requires_rdkit

